packages/trader/src/Constants/trade-categories.ts

```diff
@@ -0,0 +1,10 @@
+// Trade types that are considered as vanilla financials
+export const vanilla_financials = [
+    'frxAUDUSD',
```

Comment:
gold and silver not added ?

Reply:
Vanilla Financial is only available for forex at the moment, therefore it's not added. The only difference between this and normal VO are some specific content changes which might not apply to other assets like gold and silver, so I don't want to add it now.